mindjunkies/forums/tests/test_views.py

```
@@ -0,0 +1,369 @@
+        )
+        self.comment = ForumComment.objects.create(
+            topic=self.topic,
+            author=self.user,
+            content='Test comment'
+        )
+        self.reply = Reply.objects.create(
+            parent_comment=self.comment,
+            author=self.user,
+            body='Test reply'
+        )
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)